programmieren-lernen/dependency-injection.md

```diff
@@ -6,26 +6,32 @@ alias: ["di"]
 Dependency Injection by Example:
 ```java
 public class MeineApp {
+  // hier übergibt man die Instanz der MeineApp Klasse
   private final DepA a = new DepA(this);
-  private final Depb b = new DepB(a);
+  // die DepB Klasse greift auf die DepA Klasse zu
+  private final DepB b = new DepB(a);
+  // Info: man muss nicht die Variablen hier ↑ initialisieren, geht auch in der onEnable.
```

Comment:
Do we really want this to mention here? The variable is final, so it actually cant be initialised inside the onEnable.
This tag is also for general programming and not realted to minecraft.

Reply:
The Problem often arises in #spigot-dev, as such, it would be a good thing to mention it. Alternatively, I can put the Info inside the start method...

Reply:
Yes that would be a bit better. Maybe also mention that this only counts for minecraft for our DAUs

Reply:
SHould be solved now...